app/views/groups/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<% provide(:title, "Categories") %>
+<div class='card'>
+  <div class="border border-light shadow-sm">
+      <p class="text-muted fs-5 py-3 ps-4">All Categories:</p>
+  </div>
+  <% if @groups.empty? %>
+      <p class="text-muted fs-5 py-3 ps-4">Add your first category to start.</p>
+  <% end %>
+  <% @groups.each do |group|%>  
+      <% if group.user_id == @user %>
+        <div class="card-body container">
+          <div class="card justify-content-center shadow rounded-3">
+            <div class="row g-0">
+              <div class="col-3">
+                <img src=<%= group.icon %> class="img-fluid rounded-start p-4" alt="...">
+              </div>
+              <div class="col-6 d-flex align-items-center">
+                <div class="card-body ">
+                  <%= link_to group.name, entities_show_path(user_id: @user, id: group.id), class: 'card-title text-secondary fs-4' %>
+                  <p class="card-text mt-4 text-black-50"><%= group.created_at.strftime("%d %b %Y") %></p>
+                </div>
+              </div>
+              <div class="col-3 d-flex align-items-center">
+                <div class="card-body">
+                  <p class="card-text"><small class="fs-5 p-3">$<%= GroupEntity.where("group_id = #{group.id}").joins(:entity).sum("amount") %></small></p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+    <% end %>
+  <% end %>
```

Comment:
- [ ]  We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information